src/cloud-api-adaptor/pkg/adaptor/proxy/proxy.go

```diff
@@ -22,7 +22,7 @@ import (
 	pb "github.com/kata-containers/kata-containers/src/runtime/virtcontainers/pkg/agent/protocols/grpc"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials/insecure"
-	criapi "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+	criapi "k8s.io/cri-api/pkg/apis/runtime/v1"
```

Comment:
I guess this deserves its own commit, or if needed by this one document, in the commit message, why we are removing the alpha2.

Reply:
If we agree to remove the cri_runtime_endpoint then this commit will be restructured as we don't ned the criapi at all anymore.

Reply:
This has been removed now as I've squashed the criclient removal (which used this) into this commit.